This is a hack to address an underlying problem with vte on 11.3,
whereby fputs of an empty to string to stderr causes the window to
close.  This doesn't happen if gmake is launched from xterm nor in vte
on 11.4.

--- make-4.3/src/makeint.h.~1~	2020-01-19 20:32:59.000000000 +0000
+++ make-4.3/src/makeint.h	2020-11-25 11:29:20.817713561 +0000
@@ -806,3 +806,16 @@
 
 #define ENULLLOOP(_v,_c)   do { errno = 0; (_v) = _c; } \
                            while((_v)==0 && errno==EINTR)
+
+#ifdef fputs
+#undef fputs
+#endif
+
+inline int
+internal_fputs (const char *s, FILE *stream)
+{
+  const size_t len = strlen(s);
+  return len == 0 ? 0 : fputs(s, stream);
+}
+
+#define fputs(s, stream) internal_fputs(s, stream)
